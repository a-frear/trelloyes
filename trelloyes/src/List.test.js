import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import List from './List.js';

describe('Card component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<List cards={[]} />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const card = [
        { id: 'a', title: 'Amy', content: 'Philly' },
        { id: 'b', title: 'Tini', content: 'Tahoe' }
    ]
    const tree = renderer
    .create(<List key={1} header={'First list'} cards={card} />)
    .toJSON();
expect(tree).toMatchSnapshot();
})
});