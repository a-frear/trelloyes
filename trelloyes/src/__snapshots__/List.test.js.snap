// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders the UI as expected 1`] = `
<section
  class="List"
>
  <header
    class="List-header"
  >
    <h2>
      First list
    </h2>
  </header>
  <div
    class="List-cards"
  >
    <div
      className="Card"
    >
      <button
        type="button"
      >
        delete
      </button>
      <h3>
        Amy
      </h3>
      <p>
        Philly
      </p>
    </div>
    <div
      className="Card"
    >
      <button
        type="button"
      >
        delete
      </button>
      <h3>
        Tini
      </h3>
      <p>
        Tahoe
      </p>
    </div>
    <button
      className="List-add-button"
      type="button"
    >
      + Add Random Card
    </button>
  </div>
</section>
`;
